test/fixtures/app/controllers/comments.js

```diff
@@ -1,30 +1,30 @@
 
 export default {
 
-  * index(next) {
+  * index(ctx, next) {
     return 'comments#index'
   },
 
-  * create(next) {
+  * create(ctx, next) {
```

Comment:
'generator functions' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'next' is defined but never used.<br>'ctx' is defined but never used.